cscs-checks/prgenv/helloworld.py

```diff
@@ -130,7 +130,9 @@ def __init__(self, lang, linkage):
         self.sourcepath += '_openmp.' + lang
         self.descr += ' OpenMP ' + str.capitalize(linkage)
         self.prgenv_flags = {
-            'PrgEnv-cray': ['-homp'],
+            'PrgEnv-cray': (['-fopenmp'] if lang != 'f90' and
+                            self.current_system.name in {'daint', 'dom'} else
+                            ['-homp']),
```

Comment:
@gppezzi Would it make sense to test the Fortran codes only with the CCE9 classic version? It would simplify the tests and avoid these completely weird `if`s. And save all the explanations as well. In any case, the `PrgEnv-cray` delegates to the classic compiler for Fortran. We have several tests that will require this type of adjustment otherwise.